cli/integration_tests/run_summary/fixture-configs/all.json

```diff
@@ -0,0 +1,12 @@
+{
+  "$schema": "https://turbo.build/schema.json",
+  "pipeline": {
+    "build": {
+      "outputs": ["dist/**"],
+      "experimentalPassThroughEnv": ["LOCAL_VAR_PT"],
+      "env": ["LOCAL_VAR_DEP"]
+    }
+  },
+  "experimentalGlobalPassThroughEnv": ["GLOBAL_VAR_PT"],
+  "globalEnv": ["GLOBAL_VAR_DEP"]
+}
```

Comment:
Are these the exact same as the fixture configs in `strict_env_vars`? Don't love the copy paste, but I guess it doesn't have to block this getting out

Reply:
I feel like I edited them, but I'll check.

Reply:
I didn't; I unified this.